pkg/macvlan/macvlan.go

```diff
@@ -156,14 +157,14 @@ func configureIface(ifName string, res *current.Result) error {
 }
 
 func getDefaultRouteInterfaceName() (string, error) {
-	routeToDstIP, err := netlink.RouteList(nil, netlink.FAMILY_ALL)
+	routeToDstIP, err := util.GetNetLinkOps().RouteList(nil, netlink.FAMILY_ALL)
```

Comment:
What's the overall idea behind moving netlink ops to an util package? Would you also be using that for mocking in the tests?

Reply:
Yes, the approach would be similar to  https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/util/net_linux.go#L19-L125 and https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/util/net_linux_unit_test.go ... Let me know if you have different ideas